support/time-helpers/strptime.js

```diff
@@ -0,0 +1,483 @@
+/* jshint -W067 */
```

Comment:
couldn't we find this on npm?

Reply:
I could not find an acceptable strptime on npm.  If you can find one, I'd be happy to use it!